package.json

```diff
@@ -36,16 +36,18 @@
     "moment": "^2.17.1"
   },
   "devDependencies": {
+    "@angular/animations": "^4.0.3",
     "@angular/cli": "^1.0.0",
-    "@angular/common": "^4.0.1",
-    "@angular/compiler": "^4.0.1",
-    "@angular/compiler-cli": "^4.0.1",
-    "@angular/core": "^4.0.1",
-    "@angular/forms": "^4.0.1",
-    "@angular/http": "^4.0.1",
-    "@angular/platform-browser": "^4.0.1",
-    "@angular/platform-browser-dynamic": "^4.0.1",
-    "@angular/router": "^4.0.1",
+    "@angular/common": "^4.0.3",
+    "@angular/compiler": "^4.0.3",
+    "@angular/compiler-cli": "^4.0.3",
+    "@angular/core": "^4.0.3",
+    "@angular/forms": "^4.0.3",
+    "@angular/http": "^4.0.3",
+    "@angular/material": "^2.0.0-beta.3",
```

Comment:
This is just to show people that they can use it with angular material's `mdInput` directive.  That's probably the most likely use case for this directive, especially given the built in material theme.  I can use a plain input if you don't think there's value in the material example.